decentraland/unity-renderer
# feat: (sdk7) change crdt executor and serialization (adr-117 rev. 2)

## What does this PR change?
Please read the issue https://github.com/decentraland/sdk/issues/574

## How to test the changes?
Steps in the playground: 
1. Go to the playground with the next query params added: https://playground.decentraland.org/?&sdk-branch=feat/adr-117-revision-v2&explorer-branch=feat/sdk7-crdt-adr-117-rev-2
2. When all is ready, click on Run and it should work
3. They should work as without any parameter: https://playground.decentraland.org/?&explorer-branch=dev&sdk-branch=main
